.github/workflows/go.yml

```diff
@@ -14,7 +14,55 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ matrix.go-version }}
-    - name: Checkout code
+    - name: Checkout code to gopath
       uses: actions/checkout@v1
-    - name: Build & Test
-      run: make
\ No newline at end of file
+      with:
+        path: go/src/github.com/hyperledger-labs/fabric-chaincode-wasm
+    - name: Debug dirs and env
+      run: |
+        pwd
+        echo "###### ENV #######"
+        env
+        echo "###### GO ENV #######"
+        go env
+        echo ${GOPATH}
+      env:
+        GOPATH: /home/runner/work/fabric-chaincode-wasm/go
+    - name: Install ginkgo
+      run: |
+        go get -u github.com/onsi/ginkgo/ginkgo
+      env:
+        GOPATH: /home/runner/work/fabric-chaincode-wasm/go
+    - name: Get HLF
+      working-directory: ./../..
+      run: |
+        mkdir hyperledger
+        cd hyperledger
+        mkdir fabric
+        cd fabric
```

Comment:
You don't need to create the fabric directory

Reply:
correct, thanks for pointing out